src/regularTasks.ts

```diff
@@ -207,7 +226,7 @@ const GLOBAL_TASKS: CandyTask[] = [
     completed: () => getKramcoWandererChance() < 1,
     do: () => adv1(wanderWhere("wanderer"), -1, ""),
     sobriety: "sober",
-    canInitializeDigitize: true,
+    post: () => (digitizeInitialized = true),
```

Comment:
I feel like I'd like this function to be named and passed in, rather than setting it directly in the `post`

just, like, `const initializeDigitize = () => (digitizeInitialized = true);` and then `post: initializeDigitize`